helm/aws-load-balancer-controller/README.md

```diff
@@ -202,6 +202,7 @@ The default values set by the application itself can be confirmed [here](https:/
 | `livenessProbe`                             | Liveness probe settings for the controller                                                               | (see `values.yaml`)                                                                |
 | `env`                                       | Environment variables to set for aws-load-balancer-controller pod                                        | None                                                                               |
 | `hostNetwork`                               | If `true`, use hostNetwork                                                                               | `false`                                                                            |
+| `dnsPolicy`                                 | Set dnsPolicy if required                                                                                | `Default`                                                                          |
```

Comment:
"Default" is not the Default DNS policy, instead `ClusterFirst` is

Reply:
Indeed. I'll fix this.